docassemble/MATCSmallClaims/data/questions/small_claims.yml

```diff
@@ -157,47 +157,32 @@ code: |
 features:
   progress bar: True
 ---
-question: Are you doing well?
-yesno: user_is_well
-progress: 20
----
-question: Done with the interview.
-subquestion: |
-  % if user_is_well:
-  I am glad you are doing well.
-  % else:
-  I am sorry to hear that!
-  % endif
-progress: 100
-mandatory: True
----
 id: basic questions intro screen
 question: |
   File a small claims complaint
 subquestion: |
-  % if form_approved_for_email_filing:
-  This interview can help you complete and file court forms in 3 steps:
-  % else:
-  This interview can help you complete and download forms in 3 steps:
-  % endif
+  This will help you fill out the forms to file a small claim by conducting an interview—
+  that is, by asking questions in plain language for you to answer. It will not give you 
+  legal advice.
   
-  Step 1. Answer questions that will fill in your form for you.<br>
-  Step 2. Preview the completed form.<br>
+  More information on the small claims process is available on the [Massachusetts self-help 
+  website](https://www.mass.gov/small-claims), where you can watch videos and read about 
+  the small claims process. The website has several useful resources, including information 
+  to the statues of limitations, the Massachusetts General Laws, and the Massachusetts 
```

Comment:
Would this be information "about" the statues of limitations instead of "to"?

Reply:
Good catch!